app/src/main/java/com/example/guineapics/RecyclerViewAdapter.java

```diff
@@ -0,0 +1,43 @@
+package com.example.guineapics;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.example.guineapics.dao.GuineaPigsDAO;
+
+public class RecyclerViewAdapter extends RecyclerView.Adapter<TileViewHolder> {
+    private LayoutInflater inflater;
+    private GuineaPigsDAO pigsDAO = GuineaPigsDAO.getDAO();
+
+    //TODO: should I store the list or is DAO acceptable?
+    public RecyclerViewAdapter(Context context /*, ArrayList<GuineaPig> guineaPigs*/) {
+        this.inflater = LayoutInflater.from(context);
+    }
+
+    @NonNull
+    @Override
+    public TileViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        View itemView = inflater.inflate(R.layout.tile_blueprint, parent, false);
```

Comment:
this parent variable for example has context, and the new things should be attached to that

Reply:
so you're saying I should create an inflater from the parent's context here instead of assigning it in the Constructor?